### DISCUSSION TITLE
Question about Farming on many machine

### DISCUSSION THREAD

**Discussion by anderzheng** [Chia-Network/chia-blockchain#2246] [Q&A]
I installed chia on  one Windows and two centos, flow the guide of [Farming-on-many-machines](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines)
Windows : run fullnode + wallet + farmer + harvester + GUI,  with 20 plots on local disk
CentOS: run harvester only， with 6 plots and 3 plots each

And, I changed the harvester's log level to INFO. The log shows  the harvester on centos connect to fullnode successfully.
+------------harvester 1---------------------------------
13:21:32.956 harvester chia.plotting.plot_tools: INFO     Loaded a total of 3 plots of size 0.2969526454471634 TiB, in 0.059206247329711914 seconds
13:21:32.957 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 5e1ddc4ae7... Found 0 proofs. Time: 0.06019 s. Total 3 plots
13:21:32.958 harvester harvester_server        : INFO     -> farming_info to peer 192.168.xxx.xxx .....
13:21:43.759 harvester harvester_server        : INFO     <- new_signage_point_harvester from peer  ...  192.168.xxx.xxxx 

+------------harvester 2---------------------------------
14:29:35.371 harvester chia.plotting.plot_tools: INFO     Loaded a total of 8 plots of size 0.7918822793781146 TiB, in 0.03292489051818848 seconds
14:29:35.372 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming b6534254e2... Found 0 proofs. Time: 0.03397 s. Total 8 plots
14:29:35.373 harvester harvester_server        : INFO     -> farming_info to peer 192.168.xxx.xxx ...
14:29:43.317 harvester harvester_server        : INFO     <- new_signage_point_harvester from peer ...  192.168.xxx.xxxx 
-------------------------------------------------+

Question1: 
I can see only 2 harvester from Windows GUI (Farm Tab), which expect should be 3

Question2:
I can only see the 20 plots at the Farm tab and Plots tab, which expect to be 30.  It seems only the plots on local disk have been added.

Is there any method to test if the harvester is working good?

**Comment by anderzheng** [comment]
I can see 3 lines in "Plots Passed Filter" like 0/20, 0/6, 0/3. They are the number of plots on 3 machines.
So, I think my machines work well.
It will be better if the GUI can show the plots and harvester directly.

**Reply by joehuchina** [reply]
Same Issue Here.     At he Fullnode GUI  {farm}.  You could see the harverster peer connection there. 
But it has low function to check the total Plot number .  If you find. Let me know too. :)
